lib/haml_lint/options.rb

```diff
@@ -21,7 +21,7 @@ def parse(args)
       end.parse!(args)
 
       # Any remaining arguments are assumed to be files
-      @options[:files] = args
+      @options[:files] = args.empty? ? ["."] : args
```

Comment:
I'm curious, do you have a `.haml-lint.yml` file in the directory? It's odd that it doesn't work out of the box for you.